koku/masu/database/sql/reporting_ocpstoragevolumelabel_summary.sql

```diff
@@ -24,16 +26,37 @@ cte_values_agg AS (
         node
     FROM cte_tag_value
     GROUP BY key, report_period_id, namespace, node
-)
-, ins1 AS (
+),
+cte_distinct_values_agg AS (
+    SELECT v.key,
+        array_agg(DISTINCT v.values) as values,
+        v.report_period_id,
+        v.namespace,
+        v.node
+    FROM (
+        SELECT va.key,
+            unnest(va.values || ls.values) as values,
```

Comment:
If a null in the array is not OK, then I'd suggest this change:

```sql
unnest(va.values || '{}'::text[]) as values
```

Also, I'd suggest putting `values` in dbl-quotes `"values"`

(same for all occurrences in this PR)